### PR TITLE
Create stop loss.md

### PR DESCRIPTION
**Pull request by kaansatir**

### CODE REVIEW

File: _mental_models/stop loss.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+layout: mental-model
+name: Stop loss
+benefit: Avoiding further damage
+summary: Letting go becomes harder with more Investing
```
Comment (demiculus) [inline _mental_models/stop loss.md:5]: > Letting go becomes harder with more Investing

Change investing to exposure

File: _mental_models/stop loss.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+prerequisites:
+---
+
+#### Description
+
+Coming Soon.
```
Comment (demiculus) [inline _mental_models/stop loss.md:11]: Handle description

File: _mental_models/stop loss.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+#### Practice
+
+Coming Soon.
```
Comment (demiculus) [inline _mental_models/stop loss.md:15]: Handle practice

File: _mental_models/stop loss.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+#### [Example-1]
```
Comment (demiculus) [inline _mental_models/stop loss.md:17]: Create at least 2 examples

**Review by demiculus — CHANGES_REQUESTED** [review]
I'll approve pr with these included

**Comment by Esqarrouth** [comment]
status?

**Comment by demiculus** [comment]
@equinimox

**Comment by kaansatir** [comment]
I'm not interested in completing the article, i see no incentive for the
work or website.

summary:
1. evaluate importance of the problem for yourself.

2. ask yourself what you can afford to lose consciously in different
layers: emotional, psychological, financial, health&relationship wise???

3. try to create exit strategies. choose whether going cold turkey or
gradually would be better. how risky is the problem; should you go
pessimistic or optimistic.
use other mm's like second order thinking, prerequisites

4. create reminders for the exit somehow (calendar event, alarm, friend,
reminding object)

**Pull request closed without merging**